page-impact.hbs

```diff
@@ -0,0 +1,134 @@
+            </div>
+            <div>
+                <img width="320" height="244" src="https://i0.wp.com/advisory.sg/wp-content/uploads/elementor/thumbs/Centre-image-ohwc21w9k9l2g53u7lvundxq8fp75yhy3v3wybo40e.jpg?w=800&ssl=1">
+            </div>
+            <div>
+                <img width="320" height="244" src="https://i0.wp.com/advisory.sg/wp-content/uploads/elementor/thumbs/Explore-Google-ohwbecet6p4pm7jkncylrdb6ukjx0sdo6ir172tl18.jpg?w=800&ssl=1">
+            </div>
+        </div> 
+
+        <br>
+        <div class="flex flex-col text-center w-full">
+            <center><p class="entry-title"><font size="4.5" color="#696969">Since <strong>September 2018,</strong> Advisory's <strong>offline programming</strong> has engaged:<br></font><br></p><center></center>
+        </div>
+
+        <div class="container-impact-number">
+            <div class="xl:w-1/4 lg:w-1/2 md:w-full px-8 py-6"><center>
+                <h1 class="single-title">9,344</h1>
+                <p class="entry-title"><font size="4.5" color="#808080">Students</font></p>
+            </center></div>
+            <div class="xl:w-1/4 lg:w-1/2 md:w-full px-8 py-6"><center>
+                <h1 class="single-title">66</h1>
+                <p class="entry-title"><font size="4.5" color="#808080">Public &amp; School Events</font></p>
+            </center></div>
+        </div>
+
+
+
+        <br>
+        <div class="flex flex-col text-center w-full">
+            <center><h1 class="single-title">PARTNERSHIPS</h1></center><br>
+            <center><p class="entry-title"><font size="4.5" color="#696969">Advisory has built formal <strong>partnerships</strong> with organisations including:<br></font><br></p><center></center>
+        </div>
+
+        <div class="container-impact-picture">
+            <div> <!--class="xl:w-1/4 lg:w-1/2 md:w-full px-8 py-6" was original class-->
+                <img src="https://i0.wp.com/advisory.sg/wp-content/uploads/elementor/thumbs/NYC-nrjij2uevj1eqeyb98omrbvt5gx2wflurs6ug71d6o.jpg?w=800&ssl=1">
+            </div>
+            <div>
+                <img src="https://i0.wp.com/advisory.sg/wp-content/uploads/elementor/thumbs/YCS-nrjij3s92d2p20wy3r39btn9qusg44pl3wubxgzz0g.jpg?w=800&ssl=1">
+            </div>
+            <div >
+                <img src="https://i0.wp.com/advisory.sg/wp-content/uploads/elementor/thumbs/Inspirit-nrjij1wkop04eszoeqa06u4ck31poqi4fnjcyx2rcw.jpg?w=800&ssl=1">
+            </div>
+            <div>
+                <img src="https://i0.wp.com/advisory.sg/wp-content/uploads/elementor/thumbs/YoungNTUC-nrjij4q3973zdmvky9hvwbeqc8ntbttbg1hteqyku8.jpg?w=800&ssl=1">
+            </div>
+            <div>
+                <img src="https://i0.wp.com/advisory.sg/wp-content/uploads/2019/12/Reactor-School-Logo_small-1-e1575763848810.png?fit=150%2C58&ssl=1">
+            </div>
+        </div>

```

Comment:
Please set a fixed width for all the images here and ensure that on overflow they will form multiple rows!